cmake/ucx-config-version.cmake.in

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (C) NVIDIA Corporation. 2021.  ALL RIGHTS RESERVED.
+#
+
+# This is a basic version file for the Config-mode of find_package().
+#
+# This file sets PACKAGE_VERSION_EXACT if the current version string and
+# the requested version string are exactly the same and it sets
+# PACKAGE_VERSION_COMPATIBLE if the current version is >= requested version.
+
+set(PACKAGE_VERSION @VERSION@)
+
+if (PACKAGE_FIND_VERSION_RANGE)
+  # Package version must be in the requested version range
+  if ((PACKAGE_FIND_VERSION_RANGE_MIN STREQUAL "INCLUDE" AND PACKAGE_VERSION VERSION_LESS PACKAGE_FIND_VERSION_MIN)
+      OR (PACKAGE_FIND_VERSION_RANGE_MAX STREQUAL "INCLUDE" AND PACKAGE_VERSION VERSION_GREATER PACKAGE_FIND_VERSION_MAX)
```

Comment:
is this file copied from somewhere? seems not ucx-specifc code

Reply:
This is copy-pasted from `/usr/lib/cmake/libzip/libzip-config-version.cmake` on my system, and this file is generated by cmake https://github.com/nih-at/libzip/blob/master/CMakeLists.txt#L428-L429.

Reply:
can we auto-generate it during UCX `./configure` or `make` process?

Reply:
I searched on the internet, I didn't see any way to automatically generate it except if the project is built by cmake.

Reply:
ok